conu/backend/docker/image.py

```diff
@@ -254,6 +256,13 @@ def run_via_binary(self, run_command_instance=None, *args, **kwargs):
         run_command_instance.image_name = self.get_id()
         run_command_instance.options += ["-d"]
 
+        if volume:
+            if not isinstance(volume, list):
+                volume = [volume]
+            volumes = [Volume.create_from_tuple(v) for v in volume]
+            for v in volumes:
+                run_command_instance.options += ["-v", v.cmd_option]
```

Comment:
There is no cmd_option in Volume. 
str(v)?
```
>>> import conu
>>> cont = conu.DockerImage('fedora').run_via_binary(volume=(conu.Directory('/usr/bin'), "/mountpoint", "Z"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dhodovsk/repos/github.com/fedora-modularity/conu/conu/backend/docker/image.py", line 264, in run_via_binary
    run_command_instance.options += ["-v", v.cmd_option]
AttributeError: 'Volume' object has no attribute 'cmd_option'
```

Reply:
omg, I've forgotten to push this.